typescript-generator-gradle-plugin/src/main/java/cz/habarta/typescript/generator/gradle/GenerateTask.java

```diff
@@ -129,6 +130,8 @@ public class GenerateTask extends DefaultTask {
     public boolean jackson2ModuleDiscovery;
     public List<String> jackson2Modules;
     public Logger.Level loggingLevel;
+    public boolean disableTrailingSemiColon;
+    public Map<Class<?>, List<String>> injectCustomProperties;
```

Comment:
This task is setup in `build.gradle`, but the corresponding user-defined classes are not accessible there.
Should be class name as string - similar to other fields.

Reply:
Addressed this comment